Input.js

```diff
@@ -0,0 +1,111 @@
+export class Input {
+  constructor(inputContainer, displayLineFirst, displayLineSecond) {
+    this.inputContainer = inputContainer;
+    this.firstNumber = null;
+    this.secondNumber = null;
+    this.eqSign = null;
+    this.displayLineFirst = displayLineFirst;
+    this.displayLineSecond = displayLineSecond;
+  }
+  digitButtonPressed = (digit) => {
+    if (this.firstNumber && !this.eqSign) {
+    } else if (digit !== ".") {
+      this.inputContainer.innerHTML = this.inputContainer.innerHTML + digit;
+    } else if (this.inputContainer.innerHTML.includes(".") !== true) {
+      if (this.inputContainer.innerHTML === "") {
+        this.inputContainer.innerHTML = "0.";
+      } else {
+        this.inputContainer.innerHTML = this.inputContainer.innerHTML + digit;
+      }
+    }
+  };
+
+  actionButtonPressed = (sign) => {
+    if (
+      sign === "-" &&
+      this.inputContainer.innerHTML === "" &&
+      this.eqSign !== null
```

Comment:
put those conditions in well named variables to increase the readability

Reply:
done